runtime/src/lib.rs

```diff
@@ -104,11 +107,39 @@ pub use sp_consensus_aura::sr25519::AuthorityId as AuraId;
 #[cfg(any(feature = "std", test))]
 pub use sp_runtime::BuildStorage;
 pub use sp_runtime::{
+    MultiAddress,
     Perbill,
     Percent,
     Permill,
 };
 
+/// Handles converting a weight scalar to a fee value, based on the scale and granularity of the
+/// node's balance type.
+///
+/// This should typically create a mapping between the following ranges:
+///   - `[0, MAXIMUM_BLOCK_WEIGHT]`
+///   - `[Balance::min, Balance::max]`
+///
+/// Yet, it can be used for any other sort of change to weight-fee. Some examples being:
+///   - Setting it to `0` will essentially disable the weight fee.
+///   - Setting it to `1` will cause the literal `#[weight = x]` values to be charged.
+pub struct WeightToFee;
+impl WeightToFeePolynomial for WeightToFee {
+    type Balance = Balance;
+    fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+        // in Rococo, extrinsic base weight (smallest non-zero weight) is mapped to 1 MILLIUNIT:
```

Comment:
have you discussed these values for `p` and `q` with the team?

Reply:
No @ltfschoen . I picked it from here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.11/runtime/src/lib.rs#L139

Reply:
I have added details in shared sheet `Runtime Upgrade Values`

Reply:
instead of storing them in an internal spreadsheet, could you please create a Github issue for each of the Runtime Upgrade Values that haven't been resolved

Reply:
raised here https://github.com/DataHighway-DHX/node/issues/246